notebooks/signac_106_signac-flow_HOOMD-blue_Example.ipynb

```diff
@@ -32,19 +32,14 @@
    "metadata": {},
```

Comment:
<p>This needs to be rerun.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>